server/routes/visitJourney/checkYourBooking.test.ts

```diff
@@ -247,39 +247,45 @@ testJourneys.forEach(journey => {
+            visitService.bookVisit.mockRejectedValue({})
 
-          return request(sessionApp)
-            .post(`${journey.urlPrefix}/check-your-booking`)
-            .expect(200)
-            .expect('Content-Type', /html/)
-            .expect(res => {
-              const $ = cheerio.load(res.text)
-              expect($('h1').text().trim()).toBe('Check the visit details before booking')
-              expect($('.govuk-error-summary__body').text()).toContain('Failed to book this visit')
-              expect($('.test-prisoner-name').text()).toContain('prisoner name')
-              expect($('.test-visit-date').text()).toContain('Saturday 12 March 2022')
-              expect($('.test-visit-time').text()).toContain('9:30am to 10:30am')
-              expect($('.test-visit-type').text()).toContain('Open')
-              expect($('form').prop('action')).toBe(`${journey.urlPrefix}/check-your-booking`)
+            return request(sessionApp)
+              .post(`${journey.urlPrefix}/check-your-booking`)
+              .expect(200)
+              .expect('Content-Type', /html/)
+              .expect(res => {
+                const $ = cheerio.load(res.text)
+                expect($('h1').text().trim()).toBe('Check the visit details before booking')
+                expect($('.govuk-error-summary__body').text()).toContain('Failed to book this visit')
+                expect($('.test-prisoner-name').text()).toContain('prisoner name')
+                expect($('.test-visit-date').text()).toContain('Saturday 12 March 2022')
+                expect($('.test-visit-time').text()).toContain('9:30am to 10:30am')
+                expect($('.test-visit-type').text()).toContain('Open')
+                expect($('form').prop('action')).toBe(`${journey.urlPrefix}/check-your-booking`)
 
-              expect(visitService.bookVisit).toHaveBeenCalledWith({
-                username: 'user1',
-                applicationReference: visitSessionData.applicationReference,
-                applicationMethod: visitSessionData.requestMethod,
-                allowOverBooking: false,
-              })
+                expect(visitService.bookVisit).toHaveBeenCalledWith({
+                  username: 'user1',
+                  applicationReference: visitSessionData.applicationReference,
+                  applicationMethod: visitSessionData.requestMethod,
+                  allowOverBooking: false,
+                })
 
-              expect(visitSessionData.visitStatus).not.toBe('BOOKED')
-              expect(visitSessionData.visitReference).toBe(journey.isUpdate ? 'ab-cd-ef-gh' : undefined)
-              expect(auditService.bookedVisit).not.toHaveBeenCalled()
-            })
+                expect(visitSessionData.visitStatus).not.toBe('BOOKED')
+                expect(visitSessionData.visitReference).toBe(journey.isUpdate ? 'ab-cd-ef-gh' : undefined)
+                expect(auditService.bookedVisit).not.toHaveBeenCalled()
+              })
```

Comment:
This test above has ended up nested within the wrong `describe()` block. It should be in the containing `Handle API errors` block as it isn't a 422 error (which has its own block)